tests/bootstrap_strings.py

```diff
@@ -1,7 +1,7 @@
+TEST_MANIFEST = '# The manifest for the "dev" environment.\n# Read the full specification for the "Environment" type at:\n#  https://aws.github.io/copilot-cli/docs/manifest/environment/\n\n# Your environment name will be used in naming your resources like VPC, cluster, etc.\nname: test\ntype: Environment\n\n# Import your own VPC and subnets or configure how they should be created.\n# network:\n#   vpc:\n#     id:\n\n# Configure the load balancers in your environment, once created.\n\nhttp:\n  public:\n    certificates:\n      - ACM-ARN-FOR-test.landan.cloudapps.digital\n#   private:\n\n\n# Configure observability for your environment resources.\nobservability:\n  container_insights: true\n'
```

Comment:
I thought we were using `development` etc.?
```suggestion
TEST_MANIFEST = '# The manifest for the "development" environment.\n# Read the full specification for the "Environment" type at:\n#  https://aws.github.io/copilot-cli/docs/manifest/environment/\n\n# Your environment name will be used in naming your resources like VPC, cluster, etc.\nname: test\ntype: Environment\n\n# Import your own VPC and subnets or configure how they should be created.\n# network:\n#   vpc:\n#     id:\n\n# Configure the load balancers in your environment, once created.\n\nhttp:\n  public:\n    certificates:\n      - ACM-ARN-FOR-test.landan.cloudapps.digital\n#   private:\n\n\n# Configure observability for your environment resources.\nobservability:\n  container_insights: true\n'
```

Reply:
It's just a unit test.

Reply:
The unit test point is good thought. Do we actually need these whole long strings then or can they be short and still server the purposes of the tests?

Reply:
Admittedly, these are quite hard to read. It would be good to use multiline strings.

Reply:
So I haven't written any of this code.  And if I amend this, I'll have to amend some unit tests to make them work, which might lead to you spotting more issues, so I'm going to leave as is.  

Maybe create a PR and fix these directly?

Reply:
OK, let's look at it next time someone touches the code.